tests/bluetooth/common/testlib/include/bs_helpers/timeout.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_types.h" /* For `bs_time_t` */
+
+/* Returns the timeout value (in us) specified on the command-line. */
```

Comment:
Please elaborate about this being the "sim_length" command line parameter for this device defined in bs_helpers/timeout.c